test/e2e/gateway_api_test.go

```diff
@@ -191,6 +195,105 @@ func testGatewayAPIObjects(t *testing.T) {
 	}
 }
 
+// testGatewayAPIManualDeployment verifies that Istio's manual deployment is not
+// enabled.  When manual deployment is enabled, then Istio allows a gateway use
+// to another gateway's service by specifying that gateway's service in
```

Comment:
```suggestion
// enabled.  When manual deployment is enabled, then Istio allows a gateway use
// to another gateway's service by specifying that gateway's service in
nit:
```
```suggestion
// enabled.  When manual deployment is enabled, then Istio allows a gateway to use
// another gateway's service by specifying that gateway's service in
```

Reply:
Thanks!  Fixed in https://github.com/openshift/cluster-ingress-operator/compare/1f0444b996dfff19d6118aadb7309bc8e1293fa8..6734479ebfc5bbb83897bc51f7d7ca580f783e2b.